e2e/framework.go

```diff
@@ -193,6 +237,46 @@ func (c *cluster) WaitForHeight(num uint64, timeout time.Duration, nodes ...[]st
 	}
 }
 
+func (c *cluster) syncWithNetwork(ourselves string) (uint64, uint64) {
```

Comment:
Can we rename `ourselves` to `nodeID`? It gives the impression that the string represents something in the cluster.